_layouts/base.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+
+<!--
+  This is a basic HTML file template to become the new homepage of the site.
+  Currently it simply includes Vanilla styles and not much else.
+
+  When you come to work on it in ernest, please remove this comment
+  and replace index.html with this file.
+-->
+
+<html lang="en">
+{% include head.html %}
```

Comment:
Actually, if `base.html` is shared and is the only place that calls `head.html`, there's no need for `head.html` either.